pkg/config/reload_test.go

```diff
@@ -325,4 +326,40 @@ var _ = t.Describe("Config", func() {
 			Expect(sut.ApparmorProfile).To(Equal(profile))
 		})
 	})
+
+	t.Describe("ReloadRuntimes", func() {
+		It("should succeed without any config change", func() {
+			// Given
+			// When
+			err := sut.ReloadRuntimes(sut)
+
+			// Then
+			Expect(err).To(BeNil())
+		})
+
+		It("should fail for invalid default_runtime", func() {
+			// Given
+			newConfig := defaultConfig()
+			newConfig.DefaultRuntime = "invalid"
+
+			// When
+			err := sut.ReloadRuntimes(newConfig)
+
+			// Then
+			Expect(err).NotTo(BeNil())
+		})
+
+		It("should add a new runtime", func() {
```

Comment:
could we add a case where we verify a runtime is unchanged because it's already present?

Reply:
Sure. I will add more test cases. I wanted to get this out for early feedback.